codewof/programming/content/en/prime-numbers/question.md

```diff
@@ -0,0 +1,18 @@
+# Prime Numbers
+
+Write a program that takes a positive integer as input and **prints** all the prime numbers between 0 and that number (inclusive).
```

Comment:
To be consistent perhaps say 'asks for' rather than 'takes'

Reply:
This question has been removed, but I'll change this in case we decide to add it in the future

Reply:
It hasn't been entirely removed; perhaps for the sake of future us figuring out what exists-but-hasn't-been-added-as-a-question, the part in `questions.yaml` should be commented out rather than deleted